core/src/main/java/com/datastax/oss/driver/api/core/config/CoreDriverOption.java

```diff
@@ -49,6 +49,8 @@
   SPECULATIVE_EXECUTION_POLICY_ROOT("request.speculative-execution-policy", true),
   RELATIVE_SPECULATIVE_EXECUTION_MAX("max-executions", false),
   RELATIVE_SPECULATIVE_EXECUTION_DELAY("delay", false),
+  REQUEST_TRACE_ATTEMPTS("request.trace.attempts", true),
```

Comment:
Was just remembering that query trace can be unreliable in multi-DC scenarios ([JAVA-1473](https://datastax-oss.atlassian.net/browse/JAVA-1473)).  If we use the default consistency level (LOCAL_ONE), it's possible trace queries will fail if the replicas of the session_id are in a remote DC.  Should we considering adding a parameter to configure the trace CL, or just set it to ONE?

Reply:
Good observation, I pushed a commit to address that.